paper-tabs.html

```diff
@@ -398,30 +414,61 @@
         element.removeAttribute('noink');
       },
 
-      _computeScrollButtonClass: function(hideThisButton, scrollable, hideScrollButtons) {
+      _setScrollButtonClass: function(_leftHidden, _rightHidden, scrollable, hideScrollButtons) {
+        var scrollButtonLeftClassList = this.$.scrollButtonLeft.classList;
+        var scrollButtonRightClassList = this.$.scrollButtonRight.classList;
+
         if (!scrollable || hideScrollButtons) {
-          return 'hidden';
+          scrollButtonLeftClassList.add('hidden');
+          scrollButtonRightClassList.add('hidden');
+        } else {
+          scrollButtonLeftClassList.remove('hidden');
+          scrollButtonRightClassList.remove('hidden');
         }
 
-        if (hideThisButton) {
-          return 'not-visible';
+        if (_leftHidden) {
```

Comment:
It looks like [the bug that caused this](Polymer/polymer#3918) was fixed recently, so I'm going to try reverting all of this stuff that has to do with avoiding `class$=`. Ideally all of this will disappear... Updates in a bit.